### PR TITLE
docs: more prominent config ordering contact

### LINKED ISSUE #1176: Docs: Improve Internet Header config doc
The text about where to order a config if you don't have one is unclear and could be a little more prominent (https://next.design-system.post.ch/iframe.html?viewMode=story&id=components-internet-header-getting-started--page#configuration).

New projects need to order a config from digitalconsulting@post.ch before they can implement the header. This information needs to be more prominent so it's clear from the start. Also include a more prominent call to action with a mailto: link.

### PR DESCRIPTION
**Pull request by cococonscious**
In collaboration with @trancee and @p45q 

Fixes #1176

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/documentation/src/stories/internet-header/getting-started.stories.mdx [hunk 1]
```diff
@@ -61,9 +61,13 @@ If you are not using any loader or don't want to install from npm, you can load
 
 ## Configuration
 
-The Internet Header is built to support the already existing Portal Configuration used for the old Internet Header. If you already have a configuration, you don't have to change it. If you are working on a new project, you can order a new configuration with the [Post Portal Team](mailto:digitalconsulting@post.ch).
+The Internet Header is built to support the already existing Portal Configuration used for the old Internet Header.
 
-Not every Online Service has configurations for all environments. For example, "TOPOS" only has `int02` configured and cannot use `int01` as value for environment. If you are not sure what's configured for your project, you can contact [digitalconsulting@post.ch](mailto:digitalconsulting@post.ch).
+If you're working on a new project, you probably need a new configuration. Please click the button below.
+
+<a class="mb-small-r btn-primary btn btn-rg btn-animated" href="mailto:digitalconsulting@post.ch" target="_blank" rel="noopener"><span>Contact Post Portal Team</span></a>
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/internet-header/getting-started.stories.mdx:68]:
```suggestion
<a class="mb-small-r btn-primary btn" href="mailto:digitalconsulting@post.ch" target="_blank" rel="noopener">Contact Post Portal Team<post-icon name="1001" class="fs-large"></post-icon></a>
```

File: packages/documentation/src/stories/internet-header/getting-started.stories.mdx [hunk 2]
```diff
@@ -61,9 +61,13 @@ If you are not using any loader or don't want to install from npm, you can load
...
+
+Not every Online Service has configurations for all environments. For example, "TOPOS" only has `int02` configured and cannot use `int01` as value for environment. If you are not sure what's configured for your project, please also click the button above.
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/internet-header/getting-started.stories.mdx:70]:
```suggestion
Not every Online Service has configurations for all environments. For example, "TOPOS" only has `int02` configured and cannot use `int01` as value for environment. If you are not sure what's configured for your project, please also use the button above, to ask the post-portal team about it.
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/internet-header/getting-started.stories.mdx (hunk 2 above)
Comment (cococonscious) [reply packages/documentation/src/stories/internet-header/getting-started.stories.mdx:70]: Thanks for the suggestion, although the comma after "above" isn't really fitting. What do you think of leaving it out?

**Review by gfellerph — APPROVED** [review]

**Pull request merged**